cmd/kubeadm/app/phases/upgrade/prepull_test.go

```diff
@@ -26,6 +26,10 @@ import (
 	//"k8s.io/apimachinery/pkg/util/version"
 )
 
+const (
```

Comment:
Please move it to `cmd/kubeadm/app/constants/constants.go`

Reply:
@yagonobre Thanks for your feedback. All have been done.